addons/binding/org.openhab.binding.mihome/ESH-INF/thing/channel.xml

```diff
@@ -119,6 +119,17 @@
         <state min="0" max="100" step="0.1" pattern="%.1f %%" readOnly="true">
         </state>
     </channel-type>
+    
+    <channel-type id="pressure">
+        <item-type>Number</item-type>
+        <label>Pressure</label>
+        <category>Pressure</category>
+        <tags>
+            <tag>weather</tag>
+        </tags>
+        <state pattern="%.1f kPa" readOnly="true">
```

Comment:
The sensor natively reports in kPa as shown in the mihome app.  If other units are desirable then I'd probably wait for the unit PR to ESH.